<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 32, 44);
color: rgb(226, 232, 240);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="windowTitle">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Steganography Tool</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(209, 213, 219)</string>
    </property>
    <property name="text">
     <string>Video File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectVideo">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(8, 145, 178);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> Select</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathVideo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 65, 81);
border-radius: 5px;
padding-left: 15px;
padding-right: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHIde">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 99, 235);
border-radius:12px;</string>
    </property>
    <property name="text">
     <string> Hide Image</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertImage"/>
    </property>
   </widget>
   <widget class="QLabel" name="labelResults">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(103, 232, 249);</string>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="containerResults">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>731</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 65, 81);
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRetrieve">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(22, 163, 74);
border-radius:12px;</string>
    </property>
    <property name="text">
     <string> Retrieve Image</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonVerify">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(147, 51, 234);
border-radius:12px;</string>
    </property>
    <property name="text">
     <string> Verify Quality</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
    </property>
   </widget>
   <widget class="QLabel" name="imageLogo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
width: 300px; ;
 height: 200px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(209, 213, 219)</string>
    </property>
    <property name="text">
     <string>Image File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectImage">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(8, 145, 178);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> Select</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathImage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 65, 81);
border-radius: 5px;
padding-left: 15px;
padding-right: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
